library/Fission/Web/Middleware/AuthBody.hs

```diff
@@ -0,0 +1,39 @@
+{-# OPTIONS_GHC -fno-warn-deprecations #-}
```

Comment:
Because they don't want you normally working with `requestBody` directly, they added helpers like `strictRequestBody` and deprecated the `requestBody` accessor. It's fine, because we're using the `Internal` module since we're writing pretty surgical middleware.